### ISSUE TITLE
Why dont play directly on the VM?

### ISSUE THREAD

**Issue by davidhdezf** [ecalder6/azure-gaming#38]
Hi,

Thanks a lot for this repository I was able to set up a VM in no time. I just have one simple question. Why  is it  better to stream the game instead of using the RDP?  Does it reduce the lag?

Best regards,

David

**Comment by ecalder6** [comment]
Yep you can give RDP a a try, but it should be clear that it's not designed for game streaming (or event just video streaming).

**Comment by Stephanvs** [comment]
An alternative that I'm currently pursuing is instead of streaming the game, I've installed Parsec (available at ) and I've installed the game on the VM itself. Instead of connecting to the VM via RDP (which is not really optimized for game or video streaming) we'll use Parsec, which _is_ optimized for gaming scenarios.

So far I've been pretty happy with the setup.
